PyRDF/backend/Backend.py

```diff
@@ -30,6 +55,32 @@ def __init__(self, config={}):
         """
         self.config = config
 
+    def is_supported(self, operation_name):
```

Comment:
You need to call this from `Node`, when the user just called something on one of your nodes, [here](https://github.com/shravan97/PyRDF/blob/master/PyRDF/Node.py#L92). You do the check in two steps, first you check if the operation name you received `is_supported` by your current backend, and only if that check passes you create the `Operation` object, which will contain the classification.

Reply:
Yes @etejedor . This is a good idea